test/e2e/utils/package_plugin.go

```diff
@@ -3,6 +3,10 @@
 
 package utils
 
+func ListClusters() (string, error) {
+	return Tanzu(nil, "cluster", "list")
```

Comment:
Something for the future in a different PR - I think we can reduce the function call to just `Tanzu("cluster", "list")`. I don't see any usage of `Tanzu()` where use something other than `nil`

Similar observation for `Kubectl()`. Very few usages of `Kubectl` have the input argument other than `nil`. Maybe a separate util function can be used for those cases, with a small change in naming. For the lack of a better name, all I can think of is `KubectlWithInput`, a very verbose name

In general, the `Kubectl(nil, ..` and `Tanzu(nil, ...` usages seem unnecessary when most times the value of the argument is `nil`.